comms/db/queries/get_chats.go

```diff
@@ -42,23 +52,49 @@ func UserChat(q db.Queryable, ctx context.Context, arg ChatMembershipParams) (Us
 
 // Get all chats (with user-specific details) for the given user
 const userChats = `
+WITH last_message_per_chat AS (
```

Comment:
Thoughts on updating the chat in the mutation rather than fetching it at query time?

Reply:
that makes sense; will change that

Reply:
https://github.com/AudiusProject/audius-protocol/pull/4616/commits/8b5427526f13a9f284372bf777311bb1a4e5fcb7